Servers/manage-logs.sh

```
@@ -0,0 +1,204 @@
+        echo "Available tenants:"
+        list_tenants
+        return 1
+    fi
+
+    local tenant_dir="$LOG_BASE_DIR/$tenant_id"
+    if [ ! -d "$tenant_dir" ]; then
+        echo "Tenant '$tenant_id' not found."
+        return 1
+    fi
+
+    local latest_log=$(ls -t "$tenant_dir"/*.log 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)